web/themes/new_weather_theme/templates/partials/radar.html.twig

```diff
@@ -21,10 +21,20 @@
         });
       </script>
     </div>
-    <div class="grid-row wx-radar-container">
-       <wx-radar class="grid-col-12" lat="{{ point.lat }}" lon="{{ point.lon }}">
-         <div id="wx_radar_container" style="height: 300px;"></div>
+    <div class="grid-row wx-radar-container position-relative">
+      <wx-radar class="grid-col-12" lat="{{ point.lat }}" lon="{{ point.lon }}">
+        <div id="wx_radar_container" style="height: 300px;"></div>
       </wx-radar>
+
+      <div class="position-absolute top-0 right-0 height-full">
+        <div class="position-sticky top-0" style="top: 55px;">
```

Comment:
`top` needs to account for the height of the stickied tab list. It's about 55 pixels, so use that as a default, though the very actually real value is found and set in Javascript.